GitWorkflow/GitWorkflow/Player.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <SFML/Graphics.hpp>
+
+class Player final : public sf::Drawable
```

Comment:
For the future, we might refactor Drawables. But for now, it's fine to do it like this.

Reply:
Will refactor if needed in the future.